core/templates/dev/head/services/AlertsService.ts

```diff
@@ -16,149 +16,158 @@
  * @fileoverview Factory for handling warnings and info messages.
  */
 
-angular.module('oppia').factory('AlertsService', ['$log', function($log) {
-  var AlertsService = {
-    /**
-     * Each element in each of the arrays here is an object with two keys:
-     *   - type:  a string specifying the type of message or warning.
-     *            Possible types - "warning", "info" or "success".
-     *   - content: a string containing the warning or message.
-     */
-
-    /**
-     * Array of "warning" messages.
-     */
-    warnings: [],
-    /**
-     * Array of "success" or "info" messages.
-     */
-    messages: [],
-
-    addWarning: null,
-    fatalWarning: null,
-    deleteWarning: null,
-    clearWarnings: null,
-    addMessage: null,
-    deleteMessage: null,
-    addInfoMessage: null,
-    addSuccessMessage: null,
-    clearMessages: null
-  };
+import { Injectable } from '@angular/core';
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { LoggerService } from 'services/contextual/LoggerService';
```

Comment:
Should there be a  new line in between angular and local imports? (not sure but can you please check?)

Reply:
Done